packages/libs/react-ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,6 +1,6 @@
-import { Accordion, IAccordionProps } from './Accordion';
+import { type IAccordionProps, Accordion } from './Accordion';
 
-import type { Meta, StoryObj } from '@storybook/react';
+import { type Meta, type StoryObj } from '@storybook/react';
```

Comment:
Is there a way to avoid this update when all the imports are types?

Reply:
Yes, you can just do `import type { ... }` actually. You think that would be better?

Reply:
I like the new change better as it's more explicit. Also, when another import from the same package is added in the future that isn't a type, the changes will be easier to spot

Reply:
Agreed!

Reply:
Sounds good to me 👍